lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -435,16 +435,10 @@ static void collectFileLineColLocs(Location loc,
 
 /// Return the location information as a (potentially empty) string.
 static std::string
-getLocationInfoAsStringImpl(const SmallPtrSetImpl<Operation *> &ops) {
+getLocationInfoAsStringImpl(const SmallPtrSet<Attribute, 8> &locationSet) {
```

Comment:
nit: Plz continue to use `SmallPtrSetImpl`.

Reply:
🤦  thanks